examples/postgresql/v10.16.2/postgres-blueprint.yaml

```diff
@@ -70,7 +70,7 @@ actions:
       name: deleteDump
       args:
         image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
-        namespace: "{{ .Namespace.Name }}"
+        namespace: "{{ .StatefulSet.Namespace }}"
```

Comment:
Same as above

Also, I believe this blueprint is no longer relevant. We can remove the v10.16.2 and v8.6.4 folders completely. @viveksinghggits @PrasadG193 WDYT?

Reply:
reverted the change

Reply:
sorry, maybe I am missing some context. Why this blueprint not relevant?

Reply:
@viveksinghggits are there any specific uses for this blueprint? Do we need to support this particular version of postgres?

Reply:
I don't think we have to support that specific version. But I think it's ok to keep the stale versions of blueprint just for future reference. if someone wants to refer to specific postgres' versions blueprint.
I think this is the reason we decided to have these dirs with older blueprints right?
I am also ok if decide to remove them.